### ISSUE TITLE
Constants names

### ISSUE THREAD

**Issue by volodymyrdovhan** [rezeau/moodle-qtype_multianswerrgx#3]
Hello,
based on Moodle standards, all plugin constants should start with frankenstyle prefix.
In the plugin uses '_RGX' suffix for the constants, which should be fine to prevent naming collisions.
However, as a plugin improvement, we recommend using the frankenstyle prefix

https://moodledev.io/general/community/plugincontribution/checklist#namespace-collisions

![image](https://github.com/user-attachments/assets/c3bee785-8692-4615-a2c9-d1f75673de7f)

**Comment by rezeau** [comment]
Hi @volodymyrdovhan 
Actually I have copied these constants from the original multianswer question type [here](https://github.com/moodle/moodle/blob/a798b1db394dd41812b74540d1bed9aedab2de40/question/type/multianswer/questiontype.php#L388C1-L393C62). I only added the _RGX suffix to prevent naming collisions. If you agree to pass my question type submission as is, then I will not change these constants to the frankenstyle syntax, as I find it easier to understand how things work in my multianswerrgx qtype as compared to the original multianswer qtype.
